### PR TITLE
Allow building libjdk.a for linux-aarch64

### PR DESCRIPTION
**Pull request by jperedadnr**
Use `linux-cross-compile` job, but only for `linux-aarch64`, not other targets (arm, s390x, ppc64le, riscv64)

### CODE REVIEW

**Comment by abhinayagarwal** [comment]
can you also update the `static-release.yml` ?

**Comment by jperedadnr** [comment]
Done, I've also renamed the artifact that is built to linux-aarch64, instead of linux-cross-compile.
I've tested it successfully here:
job: https://github.com/jperedadnr/mobile/actions/runs/10708791081
release: https://github.com/jperedadnr/mobile/releases/tag/test2

File: .github/workflows/build-cross-compile.yml [hunk 1]
```diff
@@ -186,3 +197,10 @@ jobs:
           make-target: 'hotspot ${{ inputs.make-arguments }}'
           platform: linux-${{ matrix.target-cpu }}
         if: steps.create-sysroot.outcome == 'success' || steps.get-cached-sysroot.outputs.cache-hit == 'true'
+
+      - name: 'Upload bundles'
+        uses: ./.github/actions/upload-bundles
+        with:
+          platform: linux-${{ matrix.target-cpu }}
+          debug-suffix: '${{ matrix.suffix }}'
```
Comment (abhinayagarwal) [inline .github/workflows/build-cross-compile.yml]: This shouldn't be required as there is no `matrix.suffix` in this workflow. I see I have done the same mistake in android and ios builds both of which do not have a matrix.

Comment (jperedadnr) [reply .github/workflows/build-cross-compile.yml]: done

**Review by abhinayagarwal — APPROVED** [review]

**Pull request merged**